tests/integration_tests/performance/configs/test_snapshot_restore_performance_config_4.14.json

```diff
@@ -4475,14 +4475,8 @@
         "latency": {
             "statistics": [
                 {
-                    "criteria": "EqualWith",
-                    "function": "Percentile50",
-                    "name": "P50"
-                },
-                {
-                    "criteria": "EqualWith",
-                    "function": "Percentile90",
-                    "name": "P90"
+                    "function": "Average",
+                    "name": "Avg"
```

Comment:
Could we move these criteria to the test?

Reply:
Mh, yeah, it'd be possible, but the entire "measurements" key would have to be moved, so it'd be a bit noisy in the test, I think. Especially since I'd like to keep thingy consistent, and some other tests (e.g. block) have 4 different entries in that dictionary.